miasm2/jitter/llvmconvert.py

```diff
@@ -1350,16 +1356,16 @@ def from_asmblock(self, asmblock):
 
             # Pre-create basic blocks
             for irblock in irblocks:
-                self.append_basic_block(irblock.label, overwrite=False)
+                self.append_basic_block(irblock.loc_key, overwrite=False)
 
             # Generate the corresponding code
             for index, irblock in enumerate(irblocks):
                 new_irblock = self.llvm_context.ir_arch.irbloc_fix_regs_for_mode(
                     irblock, self.llvm_context.ir_arch.attrib)
 
                 # Set the builder at the begining of the correct bbl
-                name = self.canonize_label_name(new_irblock.label)
-                self.builder.position_at_end(self.get_basic_bloc_by_label(name))
+                name = self.llvm_context.canonize_label_name(new_irblock.loc_key)
```

Comment:
`name` was used to get back the BBL, it is no longer needed

Reply:
done